source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -0,0 +1,437 @@
+      :alt: Select Keyboard menu
+
+      Figure 6: Select Keyboard menu
+
+Select Installation Media
+=========================
+
+#. From the Main Menu, select :guilabel:`Select Installation Media`.
+
+#. Choose an installation method: `Safe Installation`_ or
+   `Destructive Installation`_.
+
+   .. figure:: figures/bare-metal-install-desktop-07.png
+      :scale: 100%
+      :alt: Select Installation Media
+
+      Figure 7: Select Installation Media
+
+Safe Installation
+-----------------
+Use this method to safely install |CL| on media with available space, or
+alongside existing partitions, and accept the `Default partition schema`_.
+If enough free space exists, safe installation is allowed.
+
+Destructive Installation
+------------------------
+
+Use this method to destroy the contents of the target device, install |CL| on it, and accept the `Default partition schema`_.
+
+Disk encryption
+===============
+
+For greater security, disk encryption is supported using LUKS. Encryption is
+optional.
+
+#. To encrypt the root partition, select :guilabel:`Enable Encryption`,
+   as shown in Figure 8.
+
+   .. figure:: figures/bare-metal-install-desktop-08.png
+      :scale: 100%
+      :alt: Enable Encryption
+
+      Figure 8: Enable Encryption
+
+#. When :guilabel:`Encryption Passphrase` appears, enter a passphrase.
+
+   .. figure:: figures/bare-metal-install-desktop-09.png
+      :scale: 100%
+      :alt: Encryption Passphrase
+
+      Figure 9: Encryption Passphrase
+
+   .. note::
+
+      Minimum length is 8 characters. Maximum length is 94 characters.
+
+#. Enter the same passphrase in the second field.
+
+#. Select :guilabel:`Confirm` in the dialogue box.
+
+   .. note::
+
+      :guilabel:`Confirm` is only highlighted if passphrases match.
+
+#. Select :guilabel:`Confirm` in submenu.
+
+Manage User
+===========
+
+#. In Advanced Options, select :guilabel:`Manage User`.
+
+#. In :guilabel:`User Name`, enter a user name.
+
+   .. figure:: figures/bare-metal-install-desktop-10.png
+      :scale: 100%
+      :alt: Manage User
+
+      Figure 10: Manage User
+
+#. In :guilabel:`Login`, create a login name. It must start with a letter
+   and can use numbers, hyphens, and underscores. Maximum length is 31
+   characters.
+
+#. In :guilabel:`Password`, enter a password. Minimum length is
+   8 characters. Maximum length is 255 characters.
+
+#. In :guilabel:`Confirm`, enter the same password.
+
+   .. note::
+
+      :guilabel:`Administrator` rights are selected by default.
+      For security purposes, the default user must be assigned as an
+      Administrator.
+
+#. Select :kbd:`Confirm`.
+
+   .. note::
+
+      Select :guilabel:`Cancel` to return to the Main Menu.
+
+Modify User
+-----------
+
+#. In Manager User, select :guilabel:`Manage User`.
+
+#. Modify user details as desired.
+
+#. Select :guilabel:`Confirm` to save the changes you made.
+
+   .. note::
+
+      Optional: Select :guilabel:`Cancel` to return to the Main Menu.
```

Comment:
Consider:
 `Cancel` to return to the Main Menu **without saving changes**.

Reply:
Got it.